src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -110,17 +123,140 @@ const VideoDetailPage = () => {
+                  alt="institution logo"
+                  className="mr-2 logo-cutom-style p-1"
+                />
+                <div className="x-small">
+                  <Hyperlink
+                    destination={`/${enterpriseCustomer.slug}/course/${courseMetadata?.key}`}
+                    target="_blank"
+                  >
+                    {courseMetadata?.title}
+                  </Hyperlink>
+                </div>
+              </div>
+              <VideoCourseReview courseKey={courseMetadata.key} />
+              <div className="d-flex flex-column mt-2.5">
+                <div className="d-flex flex-row align-items-center">
+                  <strong className="mr-1">
+                    <FormattedMessage
+                      id="enterprise.courseAbout.courseSidebar.price.original"
+                      defaultMessage="Original price:"
+                      description="Label for the original price of the course."
+                    />
+                  </strong>
+                  <s>${getCoursePrice(courseMetadata)} USD</s>
+                  <h4 className="text-danger ml-2 m-0">
+                    <FormattedMessage
+                      id="enterprise.courseAbout.courseSidebar.price.free"
+                      defaultMessage="FREE"
+                      description="Label for the price of the course. FREE"
+                    />
+                  </h4>
+                </div>
+                <span className="text-muted small">
+                  <FormattedMessage
+                    id="enterprise.courseAbout.courseSidebar.price.coveredBy.your.organization"
+                    defaultMessage="Covered by your organization"
+                    description="Label for the price of the course. Covered by your organization"
+                  />
+                </span>
+              </div>
+            </div>
+            <div className="my-4.5">
+              {hasTruthyValue(courseMetadata.activeCourseRun.weeksToComplete) && (
+                <div className="d-flex flex-row align-items-center mb-3">
+                  <Icon className="mr-2" size="lg" src={Timelapse} style={{ height: '40px', width: '40px' }} />
+                  <div className="d-flex flex-column p-0 m-0">
+                    <h5 className="m-0 x-small">{courseMetadata.activeCourseRun.weeksToComplete} weeks</h5>
+                    {hasTruthyValue([courseMetadata.activeCourseRun.minEffort, courseMetadata.activeCourseRun.maxEffort]) && (
+                      <span className="text-muted x-small">
+                        {`${courseMetadata.activeCourseRun.minEffort}-${courseMetadata.activeCourseRun.maxEffort} hours per week`}
+                      </span>
+                    )}
+                  </div>
+                </div>
+              )}
+              <div className="d-flex flex-row align-items-center mb-3">
+                <Icon className="mr-2" size="lg" src={Person} style={{ height: '40px', width: '40px' }} />
```

Comment:
Same as above

Reply:
Actually the width and height is fix as per figma and we don't have any defined className to fix that height and width.